src/xdocs/config_metrics.xml

```diff
@@ -197,6 +197,12 @@
                 Set, HashSet, SortedSet, TreeSet,
                 Map, HashMap, SortedMap, TreeMap</td>
           </tr>
+          <tr>
+            <td>excludeClassesRegexp</td>
+            <td>User-configured regular expression to ignore classes</td>
+            <td><a href="property_types.html#regexp">Regular Expression</a></td>
+            <td><code>"^[A-Z]$"</code></td>
```

Comment:
You did not set default value of ```excludeClassesRegexp``` in the code. Why do you state that it is ```^[A-Z]$``` here? Why the default value should be ```^[A-Z]$``` ?